src/logic/contracts/safeContracts.js

```diff
@@ -127,6 +134,49 @@ export const validateProxy = async (safeAddress: string): Promise<boolean> => {
     }
   }
   // Old PayingProxyCode
```

Comment:
This comments belongs to the other vairable

Reply:
done